test_infra/src/lib.rs

```diff
@@ -719,17 +719,26 @@ pub fn ssh_command_ip(
 }
 
 pub fn exec_host_command_status(command: &str) -> ExitStatus {
-    std::process::Command::new("bash")
-        .args(["-c", command])
-        .status()
-        .unwrap_or_else(|_| panic!("Expected '{command}' to run"))
+    exec_host_command_output(command).status
 }
 
 pub fn exec_host_command_output(command: &str) -> Output {
-    std::process::Command::new("bash")
+    let output = std::process::Command::new("bash")
         .args(["-c", command])
         .output()
-        .unwrap_or_else(|_| panic!("Expected '{command}' to run"))
+        .unwrap_or_else(|e| panic!("Expected '{command}' to run. Error: {:?}", e));
+
+    if !output.status.success() {
+        let stdout = String::from_utf8_lossy(&output.stdout);
+        let stderr = String::from_utf8_lossy(&output.stderr);
+        eprintln!(
+            "\n\n==== Start 'exec_host_command' failed ====\
+                \n\n---stdout---\n{stdout}\n---stderr---{stderr} \
```

Comment:
Formatting looks a bit strange on this multiline string - I think it would be nicer if it could be aligned.

Reply:
I agree... I had them aligned but `cargo fmt` made it this way... I am not sure if our GH action would be happy if I don't follow Cargo's lead...

Reply:
Cargo knows best - maybe the lack of space before the '\' confused it?

Reply:
Found the cause: it is required to be ` \` (a space ahead of the `\`) to break a string into multiple lines without having a line break.